core/core.go

```diff
@@ -537,14 +537,52 @@ func insertArgs(baseArgs []string, newArgs []string) []string {
 	return result
 }
 
-func shutdownIfNeeded(bazelPath string) {
+func parseStartupOptions(baseArgs []string) []string {
+	var result []string
+	var BAZEL_COMMANDS = []string {
```

Comment:
Can you turn this into a map for faster lookup?

Reply:
Done!